bCNC/Pendant.py

```diff
@@ -38,6 +38,17 @@
 class Pendant(httpserver.BaseHTTPRequestHandler):
     camera = None
 
+    # ----------------------------------------------------------------------
+    def get_file_size(fh):
```

Comment:
thanks

Reply:
Maybe it's related to this? https://bugs.python.org/issue41092

Reply:
Perhaps `os.stat(path).st_size` would give better results?

Reply:
May be... But I'am not a professional Python programmer, so I don't know exact details about that.